dll/opengl/opengl32/wgl.c

```diff
@@ -744,7 +744,10 @@ BOOL WINAPI wglMakeCurrent(HDC hdc, HGLRC hglrc)
         /* Winetest conformance */
         if (GetObjectType( hdc ) != OBJ_DC && GetObjectType( hdc ) != OBJ_MEMDC)
```

Comment:
While here, merge redundant `GetObjectType` calls?

Reply:
yes, i make this change.